gui.js

```diff
@@ -4807,7 +4807,7 @@ IDE_Morph.prototype.getURL = function (url) {
         }
         throw new Error('unable to retrieve ' + url);
     } catch (err) {
-        myself.showMessage(err);
+        myself.showMessage(err.message);
```

Comment:
Would you consider changing `err.message` to `err.toString()`, which include `err.name` in the result? 
Or, equivalently `''+err`.